apps/css-workshop/src/pages/checkbox.astro

```diff
@@ -398,101 +226,37 @@ import Layout from './_layout.astro';
           flex-direction: column;
           align-items: flex-end;'
     >
-      <label class='iui-checkbox-wrapper'>
-        <span class='iui-checkbox-label'>Half checkbox</span>
-        <input class='iui-checkbox' type='checkbox' name='indeterminate' />
-      </label>
-
-      <label class='iui-checkbox-wrapper'>
-        <span class='iui-checkbox-label'>Checkbox</span>
-        <input class='iui-checkbox' type='checkbox' checked />
-      </label>
-
-      <label class='iui-checkbox-wrapper iui-loading'>
-        <span class='iui-checkbox-label'>Loading checkbox</span>
-        <input class='iui-checkbox iui-loading' type='checkbox' disabled tabindex='-1' />
-        <div
-          class='iui-progress-indicator-radial'
-          data-iui-indeterminate='true'
-          data-iui-size='x-small'
-        >
-        </div>
-      </label>
-
-      <label class='iui-checkbox-wrapper iui-disabled'>
-        <span class='iui-checkbox-label'>Checkbox</span>
-        <input class='iui-checkbox' type='checkbox' disabled />
-      </label>
-
-      <label class='iui-checkbox-wrapper iui-disabled'>
-        <span class='iui-checkbox-label'>Half checkbox</span>
-        <input class='iui-checkbox' type='checkbox' name='indeterminate' disabled />
-      </label>
-
-      <label class='iui-checkbox-wrapper iui-disabled'>
-        <span class='iui-checkbox-label'>Checkbox</span>
-        <input class='iui-checkbox' type='checkbox' checked disabled />
-      </label>
-
-      <label class='iui-checkbox-wrapper iui-positive'>
-        <span class='iui-checkbox-label'>Checkbox</span>
-        <input class='iui-checkbox' type='checkbox' />
-      </label>
-
-      <label class='iui-checkbox-wrapper iui-negative'>
-        <span class='iui-checkbox-label'>Checkbox</span>
-        <input class='iui-checkbox' type='checkbox' />
-      </label>
-
-      <label class='iui-checkbox-wrapper iui-warning'>
-        <span class='iui-checkbox-label'>Checkbox</span>
-        <input class='iui-checkbox' type='checkbox' />
-      </label>
+      <Checkbox_ indeterminate style={{ flexDirection: 'row-reverse' }}>Half checkbox</Checkbox_>
```

Comment:
I think we should just remove these checkboxes since we don't support them in React (which means nobody is using it)

Reply:
Yeah i'll get rid of those. Should I change the "checkbox on left title" or just remove it?